pkg/skaffold/docker/logger/formatter_test.go

```diff
@@ -0,0 +1,67 @@
+/*
+Copyright 2021 The Skaffold Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package logger
+
+import (
+	"bytes"
+	"strings"
+	"sync"
+	"testing"
+
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/docker/tracker"
+	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/output"
+	"github.com/GoogleContainerTools/skaffold/testutil"
+)
+
+type mockColorPicker struct{}
+
+func (m *mockColorPicker) Pick(image string) output.Color {
+	return output.Default
+}
+
+func (m *mockColorPicker) AddImage(string) {}
+
+func TestPrintLogLine(t *testing.T) {
```

Comment:
nit: might make sense to add tests that format is as expected

Reply:
done